compiler/hash-pipeline/src/workspace.rs

```diff
@@ -18,6 +19,121 @@ use hash_ast::{
 use hash_source::{InteractiveId, ModuleId, ModuleKind, SourceId, SourceMap};
 use hash_utils::tree_writing::TreeWriter;
 
+bitflags! {
+    /// Defines the flags that can be used to control the compiler pipeline.
+    pub struct SourceStageInfo: u32 {
+        /// Indicates the source has not undergone any stages after parsing.
+        const NONE = 0;
```

Comment:
`bitflags` docs says to avoid defining zero flags

Reply:
https://docs.rs/bitflags/latest/bitflags/#zero-flags

Reply:
Yeah, I will remove it and add documentation with this to say that `is_empty()` means that the module has at-least been parsed.